features/admin.users.v1/constants/user-management-constants.ts

```diff
@@ -90,6 +90,8 @@ export class UserManagementConstants {
     public static readonly TERMINATE_ALL_USER_SESSIONS_ERROR: string = "Could not terminate all the user sessions " +
         "due to some error.";
 
+    public static readonly RESEND_CODE_REQUEST_ERROR: string = "Error occured while resending the recovery option.";
```

Comment:
"recovery option" doesn't seem very end-user friendly IMO.

```suggestion
    public static readonly RESEND_CODE_REQUEST_ERROR: string = "Error occured while resending the recovery link/code.";
```

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/7992/commits/4a24c0d041c9ba61094a8b15e9880bcf87b44611